src/content/developers/tutorials/implement-metamask-and-ens-into-website/index.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Connect your website to MetaMask and ENS"
+description: "Learn how to fetch the user's account data and possible ENS domain using JavaScript and the ethers.js library and thus making it accessible to your website."
+author: "Sebastian Supreme"
+sidebar: true
+tags: ["metamask", "ens", "ethersjs", "ethers.js", "javascript", "html", "css"]
+skill: intermediate
+published: 2022-09-23
+lang: en
+---
+
+In this article you will learn to easily implement MetaMask and Ethereum Name Service support into your website.
+This is accomplished using JavaScript and the ethers.js library.
+Please note that this tutorial is only for educational purposes and is not intended to be used commercially, but rather as a quick way to set up a makeshift implementation to test your code.
```

Comment:
disclaimers and licensing is all dealt with here https://ethereum.org/en/terms-of-use/#terms-of-use
```suggestion
```